apps/website/src/pages/docs/carousel.mdx

```diff
@@ -25,8 +25,54 @@ located below the slides, consisting of "dots" and "previous"/"next" buttons, us
 
 The currently shown slide can also be changed using the left/right arrow keys or by dragging on a touch device.
 
-This component uses a composition approach so it should be used with the provided subcomponents.
+## Usage
+
+Carousels can be controlled externally by using the properties `activeSlideIndex` and `onSlideChange`.
+
+<LiveExample src='Carousel.controlled.tsx'>
+  <AllExamples.CarouselControlledExample client:load />
+</LiveExample>
+
+The subcomponents can be used outside of Carousel. The following example shows how `Carousel.DotsList` can be used without the `Carousel` component. Clicking on the right half of the slide will advance the carousel to the next slide, whereas clicking on the left half will go to the previous slide.
+
+<LiveExample src='Carousel.onlyDots.tsx'>
+  <AllExamples.CarouselOnlyDotsExample client:load />
+</LiveExample>
 
 ## Props
 
-To be defined.
+Properties table is to be defined.
+
+<PropsTable path={frontmatter.propsPath} />
+
+## Subcomponents
+
+This component uses a composition approach so it should be used with the provided subcomponents.
+
+### Slider
+
+`CarouselSlider` is the scrollable list that should consist of `CarouselSlide` components.
```

Comment:
Shouldn't these components be `Carousel.Slider` because users see them this way?

Reply:
Yep, thank you!